packages/@react-native-windows/cli/src/generator-windows/index.ts

```diff
@@ -496,3 +536,22 @@ export async function installScriptsAndDependencies(options: {
     );
   }
 }
+function getWinAppSDKPackages(nugetVersion: string): NugetPackage[] {
+  const winAppSDKPackages: NugetPackage[] = [];
+  winAppSDKPackages.push({
+    id: 'Microsoft.ReactNative.WindowsAppSDK',
+    version: nugetVersion,
+  });
+
+  winAppSDKPackages.push({
+    id: 'Microsoft.WindowsAppSDK',
+    version: '1.0.0',
+  });
+
+  winAppSDKPackages.push({
+    id: 'Microsoft.Windows.SDK.BuildTools',
```

Comment:
I don't know if / remember why this is needed, worth checking

Reply:
It's included in all of the WASDK project files hence why we had it here, but looking deeper into it -- looks like WASDK folks added this due to VSIX template / NuGet template wizard specifics (https://github.com/microsoft/WindowsAppSDK/pull/1269#discussion_r699541277). I think we should be okay to remove.